source/shared/cpp/ObjectModel/AdaptiveBase64Util.cpp

```diff
@@ -249,6 +249,17 @@ std::string AdaptiveBase64Util::Encode(const std::vector<char>& decodedBase64)
 // Format for DataURI is data:[<MediaType>][;base64],data with MediaType and base64 being optional and data is composed of [A-Z a-z 0-9 + /] characters
 std::string AdaptiveBase64Util::ExtractDataFromUri(const std::string& dataUri)
 {
-    size_t comaPosition = dataUri.find_last_of(",");
-    return dataUri.substr(comaPosition + 1);
+    size_t commaPosition = dataUri.find_last_of(",");
```

Comment:
That is a long way to go for a big image.  If it is 100k, that is a long walk.  Since we know it is a data: scheme why can't we parse if from the front.

Reply:
This isn't the part of my change.

Reply:
Yes, just noting that it wasn't a very performant approach when originally written not a regression.